### DISCUSSION TITLE
[Feature Request]: 过滤规则问题 杜比视界

### DISCUSSION THREAD

**Discussion by iceucn** [jxxghp/MoviePilot#3492] [讨论]
### 当前程序版本

1.8.7

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

在过滤规则中，已经排除了杜比视界，但对于WEB-DL DV的资源还是未完成过滤，我查询了资料，这种也属于杜比视界的资源，不支持的设备也会泛蓝色，排除杜比视界是否应该把DV的资源也加进去呢
种子名：Tell.No.One.S01.2024.2160p.WEB-DL.DDP5.1.H265.DV-HDSWEB 

![111](https://github.com/jxxghp/MoviePilot/assets/134840291/e7268e1f-8f05-40a2-9dcb-93f3db14fdfd)

**Comment by lxm9357** [comment]
我遇见了同样的问题，这应该算bug，不是Feature Request？
简单来说就是使用“排除：杜比视界”规则时，并不会完全正确生效
比如空的web组会把关键词DV放在最后，这样就排除不了，我手动把DV移到靠前的位置就可以正确排除
规则配置如下：
![规则配置](https://github.com/user-attachments/assets/a42bb113-3b5a-453e-9bbe-7d209aabc5ee)
测试结果：
![DV最后](https://github.com/user-attachments/assets/5430d54b-0e83-4078-8cee-621dbf1abd4f)
![DV前移](https://github.com/user-attachments/assets/84f75e64-93aa-42b7-ab11-b1623faac1f0)
